Back-To-School/Assets/BackToSchool/Scripts/Interfaces/UI/IView.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine;
+
+
+namespace Assets.BackToSchool.Scripts.Interfaces.UI
+{
+    public interface IView
+    {
+        public void SetRoot(RectTransform canvas);
+        public void ShowView();
```

Comment:
Not necessary, but just notice, that there is nice 'pattern' have 2 pairs of methods: Show/Hide and Enable/Disable. First pair - async and second one - immediate.